### DISCUSSION TITLE
What's up?

### DISCUSSION THREAD

**Discussion by simon-the-shark** [visual-space/visual-editor#251] [Contributions]
What's up with the project? You've made some valid points in your manifesto, but it looks like the project kinda died abruptly. Any specific plans towards this?

**Comment by adrian-moisa** [comment]
Not dead, but not having the time to commit more here. I have a regular job + the startup. I've been making a lot of effort to go live with my flutter app. So I have to delay Editor enhanvements. We still need them and are part of the plan. No ETA.
